frontend/src/component/common/ConstraintAccordion/ConstraintAccordionView/ConstraintAccordionViewHeader/ConstraintAccordionViewHeaderInfo.tsx

```diff
@@ -34,6 +33,18 @@ const StyledHeaderWrapper = styled('div')(({ theme }) => ({
     borderRadius: theme.spacing(1),
 }));
 
+const StyledHeaderMetaInfo = styled('div')(({ theme }) => ({
+    display: 'flex',
+    alignItems: 'stretch',
+    marginLeft: theme.spacing(1),
+    [theme.breakpoints.down(710)]: {
```

Comment:
It's an unusual breakpoint. Does it exist anywhere else? I would prefer for this value to be in theme.